pkg/lhutil/lighthouse.go

```diff
@@ -20,16 +20,29 @@ package lhutil
 
 import (
 	"fmt"
-	corev1 "k8s.io/api/core/v1"
-	mcsv1a1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+	"reflect"
 
+  corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	mcsv1a1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
 
 	lhconst "github.com/submariner-io/lighthouse/pkg/constants"
 )
 
+type ServiceExportConditionReason string
+
+const (
+	ReasonEmpty                  ServiceExportConditionReason = "NA"
+	ReasonServiceUnavailable     ServiceExportConditionReason = "ServiceUnavailable"
```

Comment:
Not sure these are reasons that the hub controller uses. Seems like cut and paste carry over that could be irrelevant here